<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Batch Renamer Settings</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-system-symbolic"/>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="DragTableView" name="providersTableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>500</width>
     <height>202</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <italic>false</italic>
     <underline>false</underline>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::DragDropMode::InternalMove</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SelectionMode::ContiguousSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::PenStyle::SolidLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QComboBox" name="defaultOperatorSelect">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QComboBox" name="defaultScopeSelect">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>&lt;Last used&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Name only</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Name and extension</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Extension only</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_defaultScope">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Default renaming scope</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_defaultOperator">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Default renaming operation</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DragTableView</class>
   <extends>QTableView</extends>
   <header location="global">DragTableView/DragTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
